tokio/src/task/join_set.rs

```diff
@@ -374,6 +374,79 @@ impl<T: 'static> JoinSet<T> {
         while self.join_next().await.is_some() {}
     }
 
+    /// Awaits the completion of all tasks in this `JoinSet`, returning a vector of their results.
+    /// The results will be stored in the order they completed not the order they were spawned.
+    ///
```

Comment:
We usually write a short one-sentence summary at the top, followed by a line break and then the body.

```suggestion
    /// Awaits the completion of all tasks in this `JoinSet`, returning a vector of their results.
    ///
    /// The results will be stored in the order they completed not the order they were spawned.
```